src/csharp/Microsoft.Spark/Sql/Column.cs

```diff
@@ -743,6 +743,167 @@ public Column Over()
+        ///  is contained by the evaluated values of the arguments.  
+        /// </summary>
+        /// <param name="list">List of values to check the column against</param>
+        /// <returns>Column object</returns>
+        public Column IsIn(params long[] list){
+            return Apply("isin", list);
+        }
+
+        /// <summary>
+        ///  A boolean expression that is evaluated to true if the value of this expression 
+        ///  is contained by the evaluated values of the arguments.  
+        /// </summary>
+        /// <param name="list">List of values to check the column against</param>
+        /// <returns>Column object</returns>
+        public Column IsIn(params bool[] list){
+            return Apply("isin", list);
+        }
+        
+        /// <summary>
+        ///  A boolean expression that is evaluated to true if the value of this expression 
+        ///  is contained by the evaluated values of the arguments.  
+        /// </summary>
+        /// <param name="list">List of values to check the column against</param>
+        /// <returns>Column object</returns>
+        public Column IsIn(params short[] list){
+            return Apply("isin", list);
+        }
+        
+        /// <summary>
+        ///  A boolean expression that is evaluated to true if the value of this expression 
+        ///  is contained by the evaluated values of the arguments.  
+        /// </summary>
+        /// <param name="list">List of values to check the column against</param>
+        /// <returns>Column object</returns>
+        public Column IsIn(params float[] list){
+            return Apply("isin", list);
+        }
+        
+        /// <summary>
+        ///  A boolean expression that is evaluated to true if the value of this expression 
+        ///  is contained by the evaluated values of the arguments.  
+        /// </summary>
+        /// <param name="list">List of values to check the column against</param>
+        /// <returns>Column object</returns>
+        public Column IsIn(params double[] list){
+            return Apply("isin", list);
+        }
+        
+        /// <summary>
+        ///  A boolean expression that is evaluated to true if the value of this expression 
+        ///  is contained by the evaluated values of the arguments.  
+        /// </summary>
+        /// <param name="list">List of values to check the column against</param>
+        /// <returns>Column object</returns>
+        public Column IsIn(params decimal[] list){
+            return Apply("isin", list);
+        }      
+                
+        /// <summary>
+        ///  A boolean expression that is evaluated to true if the value of this expression 
+        ///  is contained by the evaluated values of the arguments.  
+        /// </summary>
+        /// <param name="list">List of values to check the column against</param>
+        /// <returns>Column object</returns>
+        public Column IsIn(List<string> list){
```

Comment:
Why do we expose this API when we already have params. Same for all the APIs below.

Reply:
orginally it only had the List parameters but someone asked to add in the params versions - i have taken the list ones out now